master/internal/command/command.go

```diff
@@ -123,6 +123,9 @@ func (c *command) Receive(ctx *actor.Context) error {
 	case scheduler.ResourcesAllocated, scheduler.ReleaseResources:
 		return c.receiveSchedulerMsg(ctx)
 
+		// TODO propagate command kill to subscribed logActors
+		// OR just to the event manager always?
+		// message to terminate logActor
```

Comment:
Didn't we decide on how to handle this?

Reply:
yeah, this is done. how did I leave this in :facepalm: I had to do a rebase at the end

Reply:
also https://github.com/determined-ai/determined/pull/1248#discussion_r491152479

Reply:
FWIW I find one big commit to be like a million times better when rebasing.

Reply:
> FWIW I find one big commit to be like a million times better when rebasing.

I agree, I sometimes do mash them into one if I have to go through a big rebase. When I'm reviewing I'd like to see separate commits pushed after my review with no rebase so I can exactly see what was changed.